AddressBook_/AddressBook_/UIKitExtension.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  UIKitExtension.swift
+//  AddressBook_
+//
+//  Created by 이중엽 on 2023/03/12.
+//
+
+import UIKit
+
+extension UIViewController {
+    func showAlert(title: String, message: String, dismissPresentingView: Bool = false ) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        let okButton = UIAlertAction(title: "확인", style: .cancel) { _ in
+            if dismissPresentingView {
+                self.presentingViewController?.dismiss(animated: true)
+            }
+        }
+        alert.addAction(okButton)
+        present(alert, animated: true)
+    }
+}
+
+extension UITextField {
+    func setPlaceholder(color: UIColor) {
+        guard let string = self.placeholder else {
```

Comment:
return 문이 짧을 경우 (짧은 단어 하나, 프로퍼티 하나 정도?) 한 줄로 작성해주세요!

```suggestion
        guard let string = self.placeholder else { return }
```